docs/source/markdown/podman-pull.1.md

```diff
@@ -79,12 +79,15 @@ This is a Docker specific option to disable image verification to a Docker
 registry and is not supported by Podman.  This flag is a NOOP and provided
 solely for scripting compatibility.
 
+**--override-arch**=*ARCH*
+Override the machine's default architecture of the image to be pulled. For example, `arm`.
```

Comment:
I'm finding machine confusing.  Is this the host or the container?  Can you just drop `machine's`?